pkgs/top-level/all-packages.nix

```diff
@@ -29682,7 +29682,8 @@ in
 
   pentablet-driver = libsForQt5.callPackage ../misc/drivers/pentablet-driver { };
 
-  qmk_firmware = callPackage ../development/misc/qmk_firmware {
+  qmk_cli = python3Packages.callPackage ../development/misc/qmk/cli.nix { };
+  qmk_firmware = callPackage ../development/misc/qmk/firmware.nix {
```

Comment:
```suggestion
  qmk-cli = python3Packages.callPackage ../development/misc/qmk/cli.nix { };

  qmk-firmware = callPackage ../development/misc/qmk/firmware.nix {
```

Reply:
The original names are with underscores, not hyphens.

Reply:
I think this only applies to python packages.